src/pineko/kfactor.py

```diff
@@ -9,20 +9,11 @@
 from pineappl import import_only_subgrid
 
 from . import scale_variations
+from .scale_variations import orders_as_tuple
 
 DEFAULT_PDF_SET = "NNPDF40_nnlo_as_01180"
 
 
-def factgrid(subgrid):
-    """Return the array of the factorization scales squared from a subgrid."""
-    return np.array([mu2.fac for mu2 in subgrid.mu2_grid()])
-
-
-def rengrid(subgrid):
-    """Return the array of the renormalization scales squared from a subgrid."""
-    return np.array([mu2.ren for mu2 in subgrid.mu2_grid()])
-
-
 def read_kfactor(kfactor_path):
```

Comment:
```suggestion
def read_from_file(path):
```

Reply:
done it b62249b9271626b2ab78f71235e3939676758df5